### ISSUE TITLE
Set up build system with Make

### ISSUE THREAD

**Issue by silverhook** [IFOSSLawBook/ifosslawbook#6]
To simplify building, it would make sense to retort to a `Makefile` or several.

@alpianon has one that he suggested to modify and contribute. It looks very useful.

**Comment by silverhook** [comment]
@alpianon I see you created a separate branch. Did you make sure to fork it from `pandoc-clean-up` (instead of `master`)?

**Comment by alpianon** [comment]
No, I missed this point. BTW, it is still WIP, I can easily restart and create a new branch from `pandoc-clean-up`. 
In any case, the new naming convention must be amended, avoiding the use of spaces. I commented on that in issue #4.

**Comment by silverhook** [comment]
@alpianon ping?
